brayns/network/entrypoints/InspectEntrypoint.cpp

```diff
@@ -21,10 +21,103 @@
 
 #include "InspectEntrypoint.h"
 
+namespace
+{
+class SceneInspector
+{
+public:
+    SceneInspector(brayns::Engine &engine, const brayns::Vector2f &inspectPosition)
+        : _engine(engine)
+        , _inspectPosition(inspectPosition)
+    {
+    }
+
+    brayns::InspectResult inspect()
```

Comment:
Just personal preference but I would put inspectPosition in inspect() parameters and call it just position.

(As engine is more a dependency and position the context of the function call).